test-apps/display-test-app/src/frontend/App.ts

```diff
@@ -25,7 +25,8 @@ import { dtaChannel, DtaIpcInterface } from "../common/DtaIpcInterface";
 import { DtaRpcInterface } from "../common/DtaRpcInterface";
 import { ToggleAspectRatioSkewDecoratorTool } from "./AspectRatioSkewDecorator";
 import { ApplyModelDisplayScaleTool } from "./DisplayScale";
-import { ApplyModelTransformTool } from "./DisplayTransform";
+import { ApplyModelTransformTool, ClearModelTransformsTool, DisableModelTransformsTool } from "./DisplayTransform";
+import { ApplyModelClipTool } from "./ModelClipTools.ts";
```

Comment:
surprised this worked

```suggestion
import { ApplyModelClipTool } from "./ModelClipTools";
```